weekly-updates/Maria-Eriksson/Mentoring-IS.md

```diff
@@ -1,17 +1,34 @@
 # Mentoring Sessions with Isla
 
+* [11 February 2021](#date-11-february-2021)
 * [26 January 2021](#date-26-january-2021)
 * [15 January 2021](#date-15-january-2021)
 
 **Why?** Building cool connections & supporting learning
 
 **When?** Fortnightly, Tuesdays at 11:30am
 
-**Where?** Google hangout link: TBC
+**Where?** Google hangout
 
 **For how long?** 1 hour
 
-**Next meeting** 9th February 2021
+**Next meeting** 23 February 2021
+
+## Date: 11 February 2021
+
+### What did we discuss?
+* IS gave ME feedback on one of her text extraction scripts (["Exctracting_histo_step_1_210208.ipynb"](https://github.com/MariaEriksson/GOSH-2000-2018-paper/blob/main/scripts/text-extraction/Extracting_histo_step_1_210208.ipynb)), including advice on how it can be cleaned up/streamlined.
```

Comment:
```suggestion
* IS gave ME feedback on one of her text extraction scripts (["Extracting_histo_step_1_210208.ipynb"](https://github.com/MariaEriksson/GOSH-2000-2018-paper/blob/main/scripts/text-extraction/Extracting_histo_step_1_210208.ipynb)), including advice on how it can be cleaned up/streamlined.
```